tests/integration_tests.rs

```diff
@@ -0,0 +1,384 @@
+}
+
+fn base_address() -> SocketAddr {
+    (HOST, PORT).to_socket_addrs().unwrap().next().unwrap()
+}
+
+fn peridio_cloud_api_path() -> PathBuf {
+    let path = peridio_cloud_path().join("apps/peridio_cloud_api");
+
+    if path.is_dir() {
+        path
+    } else {
+        panic!(
+            "peridio_cloud_api path ({}) is not a directory",
+            path.to_str().unwrap()
+        )
+    }
+}
+
+fn peridio_cloud_core_path() -> PathBuf {
+    let path = peridio_cloud_path().join("apps/peridio_cloud_core");
+
+    if path.is_dir() {
+        path
+    } else {
+        panic!(
+            "peridio_cloud_core path ({}) is not a directory",
+            path.to_str().unwrap()
+        )
+    }
+}
+
+fn peridio_cloud_certificate_authority_path() -> PathBuf {
+    peridio_cloud_certificate_authority_directory_path().join("ca.pem")
+}
+
+fn peridio_cloud_certificate_authority_directory_path() -> PathBuf {
+    // See https://github.com/peridio/peridio-cloud/blob/9f94887fb374f5b781cc1d37e1b8a4b59738456f/config/dev.exs#L7-L9
+    match env::var("peridio_cloud_CA_DIR") {
+        Ok(peridio_cloud_ca_dir) => PathBuf::from(peridio_cloud_ca_dir),
+        Err(var_error) => match var_error {
+            VarError::NotPresent => peridio_cloud_path().join("test/fixtures/ssl"),
+            VarError::NotUnicode(_) => {
+                panic!("peridio_cloud_CA_DIR environment variable is not valid unicode")
+            }
+        },
+    }
+}
+
+fn peridio_cloud_path() -> PathBuf {
+    let path = match env::var("PERIDIO_CLOUD_PATH") {
+        Ok(peridio_cloud_path) => PathBuf::from(peridio_cloud_path),
+        Err(var_error) => match var_error {
+            VarError::NotPresent => fs::canonicalize("../peridio-cloud").unwrap(),
+            VarError::NotUnicode(_) => {
+                panic!("PERIDIO_CLOUD_PATH environment variable is not valid unicode")
+            }
+        },
+    };
+
+    if path.is_dir() {
+        path
+    } else {
+        panic!(
+            "peridio_cloud path ({}) is not a directory",
+            path.to_str().unwrap()
+        );
+    }
+}
+
+fn random_name() -> String {
+    format!(
+        "peridio-cli-{}",
+        Uuid::now_v7()
+            .hyphenated()
+            .encode_upper(&mut Uuid::encode_buffer())
+    )
+}
+
+struct User {
+    username: String,
+    email: String,
+    prn_temp_path: TempPath,
+}
+impl User {
+    fn create(email_domain: &str) -> Self {
+        let username = random_name();
+        let email = format!("{}@{}", username, email_domain);
+        let password = "peridio-cli";
+        let prn_temp_path = NamedTempFile::new().unwrap().into_temp_path();
+
+        match peridio_cloud_core_mix_command()
+            .arg("peridio.user.create")
+            .args(&["--username", username.as_str()])
+            .args(&["--email", email.as_str()])
+            .args(&["--password", password])
+            .args(&["--prn-file", prn_temp_path.to_str().unwrap()])
```

Comment:
Same comment as before ^^/

Reply:
They need to all be `&str`
![Screenshot 2024-02-27 at 10 21 40 AM](https://github.com/peridio/peridio-cli/assets/298259/dc3d3263-0681-4a8e-ac65-709d787bedda)
![Screenshot 2024-02-27 at 10 21 46 AM](https://github.com/peridio/peridio-cli/assets/298259/56a750a9-d2f5-4529-a3fb-198554c6963d)
![Screenshot 2024-02-27 at 10 21 59 AM](https://github.com/peridio/peridio-cli/assets/298259/577f1f5c-4069-4127-9283-a51921e20fe5)

Reply:
`&` removed from in front of `[`.